src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
```

Comment:
☑️ don't mutate object or arrays - it will cause unexpected results later on. You should make copy using Object.assign or spread operator
☑️use proper names for object copy